### PR TITLE
Bumped hono version number

### PR DESCRIPTION
**Pull request by LoiLock**
Updated hono

### CODE REVIEW

File: examples/hono/package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  "version": "1.0.1",
+  "version": "1.0.2",
   "name": "@bun-examples/hono",
   "main": "src/index.js",
   "devDependencies": {
     "bun-types": "^0.0.83"
   },
   "dependencies": {
-    "hono": "^1.6.4"
+    "hono": "^2.0.0"
```
Comment (xHyroM) [inline examples/hono/package.json:9]:
```suggestion
    "hono": "latest"
```

**Review by xhyrom — CHANGES_REQUESTED** [review]
- (miss)

File: examples/hono/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "version": "1.0.1",
+  "version": "1.0.2",
   "name": "@bun-examples/hono",
   "main": "src/index.js",
   "devDependencies": {
     "bun-types": "^0.0.83"
```
Comment (xhyrom) [inline examples/hono/package.json]: Can you set `bun-types` to latest?

```suggestion
    "bun-types": "latest"
```

**Review by xhyrom — CHANGES_REQUESTED** [review]

File: examples/hono/package.json (hunk 2 above)
Comment (LoiLock) [reply examples/hono/package.json]: 👍🏻

**Review by xhyrom — APPROVED** [review]

**Pull request merged**